File: New folder/New folder/create-dirs.js
const fs = require('fs');
const path = require('path');

const directories = [
  'e:\\GitHub Repo\\portfolio',
  'e:\\GitHub Repo\\portfolio\\public',
  'e:\\GitHub Repo\\portfolio\\src',
  'e:\\GitHub Repo\\portfolio\\src\\components'
];

try {
  directories.forEach(dir => {
    if (!fs.existsSync(dir)) {
      fs.mkdirSync(dir, { recursive: true });
      console.log(`✓ Created: ${dir}`);
    } else {
      console.log(`✓ Already exists: ${dir}`);
    }
  });
  console.log('\n✓ All directories created successfully!');
} catch (error) {
  console.error('Error creating directories:', error.message);
}

File: New folder/New folder/create_dirs.py
#!/usr/bin/env python3
import os
import sys

base_path = r'e:\GitHub Repo\portfolio'

# Create directories
dirs = [
    os.path.join(base_path, 'src', 'components'),
    os.path.join(base_path, 'public')
]

for dir_path in dirs:
    os.makedirs(dir_path, exist_ok=True)
    print(f"✓ Created: {dir_path}")

# Create package.json
package_json = r"""{
  "name": "portfolio",
  "version": "1.0.0",
  "description": "A professional React portfolio website",
  "private": true,
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}"""

with open(os.path.join(base_path, 'package.json'), 'w') as f:
    f.write(package_json)
print(f"✓ Created: package.json")

print("\nDirectory structure created successfully!")
